website/docs/r/search_index.html.markdown

```diff
@@ -0,0 +1,243 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: search index"
+sidebar_current: "docs-mongodbatlas-resource-search-index"
+description: |-
+Provides a Search Index resource.
+---
+
+# mongodbatlas_search_index
+
+`mongodbatlas_search_index` provides a Search Index resource. This allows indexes to be created.
+
+## Example Usage
+
+```hcl
+resource "mongodbatlas_search_index" "test" {
+  name   = "project-name"
+  project_id = "<PROJECT_ID>"
+  cluster_name = "<CLUSTER_NAME>"
+  
+  analyzer = "lucene.standard"
+  collectionName = "collection_test"
+  database = "database_test"
+  mappings_dynamic = true
+  
+  searchAnalyzer = "lucene.standard"
+}
+```
+
+## Argument Reference
+
+* `name` - (Required) The name of the search index you want to create.
+* `project_id` - (Required) The ID of the organization or project you want to create the search index within.
+* `cluster_name` - (Required) The name of the cluster where you want to create the search index within.
+
+
+
+~> **NOTE:** Atlas limits the number of users to a maximum of 100 teams per project and a maximum of 250 teams per organization.
+* `analyzer` - [Analyzer](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/#std-label-analyzers-ref) to use when creating the index. Defaults to [lucene.standard](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/standard/#std-label-ref-standard-analyzer)
+
+* `analyzers` - [Custom analyzers](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/custom/#std-label-custom-analyzers) to use in this index (this is an array of objects).
```

Comment:
This needs an example

Reply:
the examples for `analyzers` are below this part